storagenode/gracefulexit/chore.go

```diff
@@ -67,6 +68,7 @@ func (chore *Chore) Run(ctx context.Context) (err error) {
 		}
 
 		for _, satellite := range satellites {
+			mon.Counter("satellite_gracefulexit_request").Inc(1) //locked
```

Comment:
could also be worth having `mon.IntVal("satellite_num_gracefulexits").Observe(len(satellites))` right after the `ListGracefulExits` call above.

Reply:
Oh and you can also do `.Inc(len(satellites))` instead of incrementing inside the loop. I didn't think about that before.